PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+  - **manual**
+  - **semi-automatic**
+  - **semi-automatic, changes must be confirmed by another user**   
+  - **automatic**
+
+
+- *ID*   
+  Product identification number. The ID number is automatically assigned by the system.
+
+- *Created on*   
+  Date and time of the creation. In the *Create product* view, no date is displayed.
+
+- *Created by*   
+  Name and username of the user who created the product. In the *Create product* view, no name is displayed.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click this button to create a new offer. The [Add to a single connection ![Add to a single connection](../../Assets/Icons/Document.png "[Single connection]") ] button and the [Add to multiple connections ![Multiple connections](../../Assets/Icons/Documents.png "[Multiple connections]") ] button are displayed.
+
+
+### Add an offer in PIM
+*PIM > Products > Tab LIST > Button Add > Button CREATE > Tab Offers > Button Add*
+
+![Add](../../Assets/Screenshots/PIM/Products/List/Offers/AddCreate.png "[Add]")
+
+- [Add to a single connection ![Document](../../Assets/Icons/Document.png "[Document]") ]    
+  Click this button to add a single variant to the product. The *Add to a single connection* window is displayed.
+
+- [Add to multiple connections ![Documents](../../Assets/Icons/Documents.png "[Documents]") ]    
+  Click this button to add multiple variants to the product. The *Add to multiple connections* wizard window is displayed.
+
+- ![Cancel](../../Assets/Icons/Cross01.png "[Cancel]") (Cancel)   
+  Click this button to cancel the connection creation and close the current view.
+
+
+### Add to a single connection
+*PIM > Products > Tab LIST > Button Add > Button CREATE > Tab Offers > Button Add > Button Add to a single connection*
+
+![Add to a single connection](../../Assets/Screenshots/PIM/Products/List/Offers/SingleConnection.png "[Add to a single connection]")
+
+**Create Offer**
+
+- *Select Connection*   
+  Click the drop-down list to select a connection for the offer. All available connections are displayed in the list.
+
+- *Attribute Set*   
+  Click the drop-down list to select an attribute set for the offer. All active attribute sets are displayed in the drop-down list. The drop-down list is locked if no connection is selected.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Generate SKU*   
+  Enable this toggle to automatically generate a SKU for the offer. Disable the toggle to enter a SKU manually. By default, this toggle is enabled.
+
+- *SKU*    
+  Enter an individual SKU for the offer. The SKU (Stock Keeping Unit) is an identification number and should be assigned only once.
+
+[comment]: <> (offer or product? how does this work? When I enter a different SKU, The SKU is automatically overwritten)
+
+- *Change tracking mode*   
+  Select the appropriate option for change tracking (ETL mode). The following options are available:   
```

Comment:
vgl. unten (multiple connections): Select the appropriate change tracking mode
Vereinheitlichen?